lib/scsi_defs.h

```diff
@@ -82,6 +82,32 @@ namespace scsi_defs {
     uint8_t vendor_specific : 2;
   };
 
+  // SCSI Reference Manual Table 202 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct TestUnitReadyCommand {
+    OpCode op_code = OpCode::kTestUnitReady;
+    uint32_t reserved : 32;;
```

Comment:
Bitfield annotations aren't necessary if you're using native sizes (8, 16, 32, 64 bits).

Reply:
We actually had a discussion about this on this thread https://github.com/googleinterns/scsi2nvme/pull/28#pullrequestreview-418531627
This was partly for style reasons

Reply:
Ok, sounds good.